zerver/migrations/0480_remove_archivedmessage_edit_history_and_more.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.2.5 on 2023-10-06 02:36
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0479_backfill_edit_history"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="archivedmessage",
+            name="edit_history",
+        ),
+        migrations.RemoveField(
+            model_name="message",
+            name="edit_history",
+        ),
+    ]
```

Comment:
This last migration can also set `edit_history_entries` to `NOT NULL`, just before removing the legacy field; that implicitly will verify that we've backfilled every entry before dropping the old column.

(Obviously we need to change `models.py` too).